tools/deployment/make_windows_package.ps1

```diff
@@ -0,0 +1,109 @@
+# We make heavy use of Write-Host, because colors are awesome. #dealwithit.
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSAvoidUsingWriteHost", '', Scope="Function", Target="*")]
+param()
+
+function Main() {
+  if (-not (Get-Command 7z.exe)) {
+    Write-Host '[-] 7z note found!  Please run .\tools\make-win64-dev-env.bat before continuing!' -ForegroundColor Red
+    exit
+  }
+
+  if ($PSVersionTable.PSVersion.Major -lt 5) {
+    Write-Host '[-] Powershell 5.0 or great is required for this script.' -ForegroundColor Red
+    exit
+  }
+
+  $version = git describe --tags
+  $latestFullVersion = (git tag)[-2]
+  # If split len is greater than 1, this is a pre-release. Chocolatey is particular
+  # about the format of the version for pre-releases.
+  if ($version.split('-').length -eq 3) {
+    $version = $version.split('-')[0] + '-' + $version.split('-')[2]
+  }
+
+  # TODO: Hash this out for more robust directory awareness
```

Comment:
I'll remove the TODO not, I wasn't entirely satisified with making assumptions with where the script is being run, however I added the `if` check for pathing after and never removed this.